src/i18n.c

```diff
@@ -12,11 +12,12 @@
 
 i18n_info i18n_map[] = {
     // Must be first, used as default
-    {"en_US", "Activate "SYSTEM_NAME, "Go to Settings to activate "SYSTEM_NAME"."},
+    {"en_US", "Activate "SYSTEM_NAME, "Go to Settings to activate " SYSTEM_NAME "."},
```

Comment:
Can you please remove spaces here and in 20th line? There is no need to put em on random places ^^

Reply:
#107 A formatter is on coming so I think it's ok